node/src/components/rpc_server/rpcs/docs.rs

```diff
@@ -101,7 +101,7 @@ pub trait DocExample {
     fn doc_example() -> &'static Self;
 }
 
-/// The main schema for the casper node's RPC server, compliant with https://spec.open-rpc.org.
+/// The main schema for the casper node's RPC server, compliant with <https://spec.open-rpc.org.>
```

Comment:
Thanks for squashing the warning, but we probably don't want the trailing period included in the link.  Also, we usually handle links like this:
```suggestion
/// The main schema for the casper node's RPC server, compliant with
/// [the OpenRPC Specification](https://spec.open-rpc.org).
```